storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2156,11 +2162,114 @@ static bool innodb_init_param(void) {
   return (TRUE);
 }
 
+/** Scan the sdi id from DD table "mysql.tables"
```

Comment:
everywhere (except variable names) use SDI instead of sdi because SDI is an abbreviation for Serialized Dictionary Information

Reply:
fixed